src/core/common/logging.hpp

```diff
@@ -483,6 +485,64 @@ extern "C" {
 #define otLogDebgBbr(...)
 #endif
 
+/**
+ * @def otLogCritMlr
+ *
+ * This method generates a log with level critical for the Multicast Listener Registration (MLR) region.
+ *
+ * @param[in]  ...  Arguments for the format specification.
+ *
+ */
+
+/**
+ * @def otLogWarnMlr
+ *
+ * This method generates a log with level warning for the Multicast Listener Registration (MLR) region.
+ *
+ * @param[in]  ...  Arguments for the format specification.
+ *
+ */
+
+/**
+ * @def otLogInfoMlr
```

Comment:
```suggestion
 * @def otLogNoteMlr
```

Reply:
I copied from `otLogInfoBbr`. So there should be issue there too.